xwiki-platform-core/xwiki-platform-legacy/xwiki-platform-legacy-extension-handler-xar/src/main/aspect/org/xwiki/extension/xar/job/diff/DocumentUnifiedDiffAspect.aj

```diff
@@ -0,0 +1,42 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.extension.xar.job.diff;
+
+/**
+ * Add a backward compatibility layer to the {@link DocumentUnifiedDiff} class.
+ *
+ * @version $Id$
```

Comment:
No since?

Reply:
Not really needed for aspects, is it?

Reply:
The aspect is not always introduced on the same version as the deprecation.

Reply:
Sure, just that I'm not entirely sure about the value it provides. Now I can add it if you prefer.

Reply:
Mainly uniformity since it's available on all the other aspects afaics. I'm not sure of the added value I agree.